packages/mui-material-next/src/MenuItem/MenuItem.js

```diff
@@ -4,17 +4,19 @@ import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { unstable_composeClasses as composeClasses } from '@mui/base/composeClasses';
 import { alpha } from '@mui/system';
+import { useSlotProps } from '@mui/base';
+import { rootShouldForwardProp } from '@mui/material/styles/styled';
+import { styled, useThemeProps } from '@mui/material/styles';
```

Comment:
Same as in `Menu.tsx`, let's change these to be imported from within `material-next`